sdk/communication/Azure.Communication.Email/src/CommunicationEmailModelFactory.cs

```diff
@@ -1,23 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-#nullable disable
-
-using Azure.Communication.Email.Models;
-using Azure.Core;
-
-namespace Azure.Communication.Email
-{
-    /// <summary> Model factory for read-only models. </summary>
-    [CodeGenModel("CommunicationEmailModelFactory")]
-    public static partial class CommunicationEmailModelFactory
```

Comment:
Curious why the model factory was removed? Given the azure SDK guideline - https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-mocking

Reply:
I've updated and made this change.

Reply:
Updated.